sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -160,6 +160,17 @@ foreach (FormPage page in formPages)
             Console.WriteLine($"    Cell ({cell.RowIndex}, {cell.ColumnIndex}) contains text: '{cell.Text}'.");
         }
     }
+
```

Comment:
Do we want to show an example of `asSelectionMarkState` usage in one of the samples?
Sorry if it's already there and I have missed it..

Reply:
What is `asSelectionMarkState` ? not sure I see the equivalent in .NET

Reply:
here - https://github.com/Azure/azure-sdk-for-net/pull/15248/files#diff-8e44b539efa928d1579e9e5d814789c9R375

Reply:
Ahh I see. So the only sample we currently showcase the other `asXXtype` is strongly typed `Receipt` sample. 
I don't see a receipt normally having selection marks. it is usually output data.
For now I will say no.
If customers ask for samples on how to use it then we can consider it